PicPi/UICollectionView+PicPi.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UICollectionView+PicPi.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 1/2/21.
+//
+
+ import UIKit
+
+extension UICollectionView {
+  
+  /// Register the given cell type using the type's description for the cell's reusable identifier.
+  func register<Cell: UICollectionViewCell>(_: Cell.Type) {
+    print(String(describing: Cell.self))
```

Comment:
nit: remove print since this would clutter the debugger

Reply:
forgot to remove it lol, thanks for checking